frontend/src/APIClients/EntityAPIClient.ts

```diff
@@ -1,4 +1,4 @@
-import baseAPIClient from "./BaseAPIClient";
+//import baseAPIClient from "./BaseAPIClient";
```

Comment:
we can probably delete all of these files related to entity? i don't think we use it anywhere